darkmode-audit/color-centralization-audit.md

```
@@ -0,0 +1,365 @@
+
+### 1f. Text / foreground greys
+| Value | file:line |
+|---|---|
+| `#d1d1d1` | 233, 238, 242, 246, 250 (5×) |
+| `#bbb` | 137, 287, 321, 1168 |
+| `#bbbbbb` | 279 |
+| `#c5c5c5` | 404, 428, 1348 |
+| `#c8c8c8` | 262, 283 |
+| `#c3c3c3` | 891 |
+| `#d6d6d6` | 1003, 1032 |
+| `#d0…` misc: `#a0a0a0` 1081, `#a5a5a5` 267, `#9a9a9a` 417, `#999` 921, `#8a8a8a` 1002, `#8b8b8b` 1307, `#777` 767/1023/1027/1990, `#707070` 2098, `#646464` 266, `#b2bfca` 1685, `#8194a5` 1681, `#8b949e` 153, `#e0dfdf` 1422, `#e6ebed` 631, `#f0f6fc` 70/367/379/385 |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)